### DISCUSSION TITLE
Rancher Desktop Design - Tray Icon

### DISCUSSION THREAD

**Discussion by rak-phillip** [rancher-sandbox/rancher-desktop#696] [Archive]
## Tray Icon

* Provide quick access to different areas by replacing Preferences with a menu item for each top-level navigation for Rancher Desktop

![Menu Bar](https://user-images.githubusercontent.com/835961/134975854-a612d4aa-417a-4015-8b02-567a82fb719f.png)

> Using the space to share this initial design proposal from the UI group. I've add some quick notes for each section of Rancher Desktop that might change. If you have any questions about anything you see below, please ask and I'll provide some more detail. 
>
> Original Discussion #683
>
> Feedback is highly valued and welcome.

**Comment by mook-as** [comment]
I suspect the menu should be native (I don't recall seeing blue-on-white menus by default in either mac or Windows).

`About` is probably not a very commonly used item; having it in the tray icon menu seems like it would be a net negative (making it infinitesimally harder to select the correct entry).

The other entries will probably depend on the results of discussion on the top-level reorg.

**Reply by rak-phillip** [reply]
You are correct, the menu would be native to the OS. I think the main takeaway here would be, in a sense, destructuring `Preferences` into the individual top-level nav items. 

I think it makes sense to drop `About` from here as well.
